plugins/scaffolder/api-report.md

```diff
@@ -464,6 +482,11 @@ export interface ScaffolderUseTemplateSecrets {
   setSecrets: (input: Record<string, string>) => void;
 }
 
+// @public (undocumented)
+export const selectedTemplateRouteRef: SubRouteRef<
```

Comment:
Is it worth mentioning these new exports in the changeset too?

Reply:
Yep, missed this !